packages/babel-plugin-transform-pipeline-operator/test/fixtures/pipeline-operator/arrow-functions/expected.js

```diff
@@ -0,0 +1,15 @@
+var _ref, _ref2, _sum;
+
+var result = (_ref = [5, 10], (_ref2 = _ref.map(x => x * 2), (_sum = _ref2.reduce((a, b) => a + b), _sum + 1)));
```

Comment:
I'm definitely overthinking this, but I wonder if it would be good to "clean up" the refs after the expression is evaluated.

Say this is an outer function that calls a long lived function (e.g. an event loop); these live references will forever prevent GC of the temporary arrays.

This idea, even if feasible, is probably out of the scope of this PR, though.

Reply:
Any idea of how we would clean up? Do we do anything like this in any other transform, since I don't know of it?

Reply:
I don't think there's a need.